core/controllers/subscriptions.py

```diff
@@ -21,26 +21,51 @@
 from core.controllers import base
 from core.domain import subscription_services
 from core.domain import user_services
+import feconf
 
 
 class SubscribeHandler(base.BaseHandler):
     """Handles operations relating to new subscriptions."""
 
+    POST_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'creator_username': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            }
+        }
+    }
+
     @acl_decorators.can_subscribe_to_users
     def post(self):
         creator_id = user_services.get_user_id_from_username(
-            self.payload.get('creator_username'))
+        self.normalized_payload.get('creator_username'))
         subscription_services.subscribe_to_creator(self.user_id, creator_id)
         self.render_json(self.values)
 
 
 class UnsubscribeHandler(base.BaseHandler):
     """Handles operations related to unsubscriptions."""
 
+    POST_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'creator_username': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            }
+        }
+    }
+
     @acl_decorators.can_subscribe_to_users
     def post(self):
         creator_id = user_services.get_user_id_from_username(
-            self.payload.get('creator_username'))
+        self.normalized_payload.get('creator_username'))
         subscription_services.unsubscribe_from_creator(
-            self.user_id, creator_id)
+        self.user_id, creator_id)
```

Comment:
Why is this changed?

Reply:
I thought of aligning it on the same line.I have now roll backed to  it's previous commit.